html/includes/table/address-search.inc.php

```diff
@@ -81,14 +83,12 @@
     $speed = humanspeed($interface['ifSpeed']);
     $type  = humanmedia($interface['ifType']);
 
-    if ($_POST['search_type'] == 'ipv6') {
-        list($prefix, $length) = explode('/', $interface['ipv6_network']);
-        $address               = Net_IPv6::compress($interface['ipv6_address']).'/'.$length;
+    if ($_POST['search_type'] == 'mac') {
```

Comment:
shit, yeah, I started to change that, then changed it back (apparently only partially)